### ISSUE TITLE
docs: update the manual backup procedure to use `DESC SCHEMA WITH INTERNALS` instead of `DESC SCHEMA`

### ISSUE THREAD

**Issue by piodul** [scylladb/scylladb#18456]
The current documentation page "Backup your data" suggests running the `DESC SCHEMA` statement in order to generate CQL commands that can be used later to restore the schema. The CQL commands generated by `DESC SCHEMA` do not contain some details that might be necessary to restore schema in some cases. Our current goal for 6.0 is to polish `DESC SCHEMA WITH INTERNALS` so that it can be used instead. Therefore, we should update the documentation to suggest using `DESC SCHEMA WITH INTERNALS` instead of just `DESC SCHEMA`.

It's also worth mentioning that this procedure requires proper support in cqlsh. The current implementation intercepts both `DESC SCHEMA` and `DESC SCHEMA WITH INTERNALS` and handles them in a special way, leading to wrong results for the latter query. This is already tracked in https://github.com/scylladb/scylla-cqlsh/issues/17. Because using an old cqlsh version will lead to an incorrect schema dump which might not be sufficient to restore the schema later, it's very important to inform the users that they must use the right version of cqlsh (and tell them to check it before doing backup).

**Comment by Jadw1** [comment]
> It's also worth mentioning that this procedure requires proper support in cqlsh. The current implementation intercepts both `DESC SCHEMA` and `DESC SCHEMA WITH INTERNALS` and handles them in a special way, leading to wrong results for the latter query. This is already tracked in [scylladb/scylla-cqlsh#17](https://github.com/scylladb/scylla-cqlsh/issues/17). Because using an old cqlsh version will lead to an incorrect schema dump which might not be sufficient to restore the schema later, it's very important to inform the users that they must use the right version of cqlsh (and tell them to check it before doing backup).

Is server-side describe used by Scylla and scylla-cqlsh at all?

Here in scylla-cqlsh: https://github.com/scylladb/scylla-cqlsh/blob/master/bin/cqlsh.py#L1593, cqlsh will do client-side describe when `connection_versions['build'][0] < '4'` and server-side describe otherwise.
But in Scylla: https://github.com/scylladb/scylladb/blob/master/version.hh#L27, version value is `3.0.8`. `connection_versions` in cqlsh **is not** a the Scylla version (like upcoming 6.0).

@avelanarius I think we discussed a some time ago all requirements to bump up `connection_versions` and maintain compatibility with cassandra.
Cc @fruch

**Comment by piodul** [comment]
Cqlsh with support for server-side describe was released this Monday: https://github.com/scylladb/scylla-cqlsh/releases/tag/v6.0.19-scylla. We are no longer blocked by this.

**Comment by piodul** [comment]
The PR was merged (https://github.com/scylladb/scylladb/commit/5b4e688668bf47d5006549471c6c18d77404544c) and backported to 6.0 (https://github.com/scylladb/scylladb/commit/803662351d7a0757dda36cb2c54b60ac1db61c60).

Closing manually because the commit was missing a "Fixes" annotation.
